### PR TITLE
Updating the report suite section in the documentation

### PR DESCRIPTION
**Pull request by rokwon**
I still need to update the screenshot.

### CODE REVIEW

**Comment by rekrishn** [comment]
Hi Robert,

What is your GH user name? I can add you as a collaborator so that you can assign me as a reviewer going forward.

Thanks,
Rekha

File: using-mobile-extensions/adobe-analytics/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 Want to know how to find report suite IDs? See [Report Suites](https://docs.adobe.com/content/help/en/analytics/admin/manage-report-suites/report-suites-admin.html).
 {% endhint %}
 
-Type one or more report suite identifiers to which the Analytics data should be sent. To add multiple report suite IDs, click **Add Another**; to remove these IDs, click **Remove Circle.** Report suite IDs can also be configured for the Development and Staging environments.
+Select one or more report suite identifiers from the combo box to which the Analytics data should be sent.  To remove a report suite ID, click on the **X** next to the ID to be removed.  Report suite IDs can also be configured for the Development and Staging environments.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/README.md]: please reword the end of the 2nd sentence to read as follows:

"...click on the X next to the ID that you want to remove."

**Comment by rokwon** [comment]
@rekrishn I've updated the screenshot now.

**Comment by rokwon** [comment]
Let's hold off merging until the UI has been released to the public :)

**Comment by rokwon** [comment]
@rekrishn Can you review my latest commit please?

**Comment by rokwon** [comment]
@rekrishn made the changes you requested :)

**Pull request merged**